docs/src/pages/[platform]/connected-components/storage/storageimage/react.mdx

```diff
@@ -43,15 +43,41 @@ At a minimum you must include the `alt`, `imgKey` and `accessLevel` props. `acce
   </ExampleCode>
 </Example>
 
+<Accordion.Container>
+  <Accordion.Item key={"title"} value={"title"}>
+    <Accordion.Trigger>
+      Using `@aws-amplify/ui-react-storage` version 3.0.1x or below?
```

Comment:
We probably need to come back and revisit this version number when we know exactly what it will be. Right now it's 3.0.18

Reply:
Ok I can make it 3.0.18 for now and update as needed